config/crd/bases/k8s.nginx.org_nginxingresscontrollers.yaml

```diff
@@ -197,6 +197,13 @@ spec:
                     minimum: 1023
                     nullable: true
                     type: integer
+                  secret:
+                    description: A Secret with a TLS certificate and key for TLS termination
```

Comment:
This description (and the one in bundle) doesn't match the comment in `types`. I think you need to run `make bundle` again.

Reply:
Ah I didn't realise it was generated. I ran `make bundle` and got the following error:
make bundle
```
/Users/soneill/go/src/github.com/nginxinc/nginx-ingress-operator/bin/controller-gen "crd:trivialVersions=true,preserveUnknownFields=false" rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
operator-sdk generate kustomize manifests -q
FATA[0000] failed to read config: error unmarshalling project configuration: error unmarshaling JSON: while decoding JSON: json: cannot unmarshal array into Go struct field Config.layout of type string
make: *** [bundle] Error 1
```
I've update this PR anyway with he generated changes.

Reply:
The files in bundle are auto-generated too.